spec/models/podcast_episode_spec.rb

```diff
@@ -116,8 +116,14 @@
   end
 
   context "when callbacks are triggered after save" do
+    let(:podcast_episode) { build(:podcast_episode) }
+
+    before do
+      allow(PodcastEpisodes::BustCacheWorker).to receive(:perform_async)
+    end
     it "triggers cache busting on save" do
-      expect { build(:podcast_episode).save }.to have_enqueued_job.on_queue("podcast_episodes_bust_cache")
+      podcast_episode.save
+      expect(PodcastEpisodes::BustCacheWorker).to have_received(:perform_async).with(podcast_episode.id, podcast_episode.path, podcast_episode.podcast_slug)
```

Comment:
I think here you can just check if the job is queued, with `sidekiq_assert_enqueued_with`, see https://github.com/thepracticaldev/dev.to/issues/5305#issuecomment-570264567 :)

Reply:
thanks @rhymes, i will change it